generators/client/templates/react/src/main/webapp/app/app.css

```diff
@@ -41,6 +42,53 @@ a:hover {
    font-weight: bold;
 }
 
+.pagination li a {
```

Comment:
why are these styles needed?

Reply:
it was useful to override the pagination with react-bootstrap

Reply:
react-bootstrap is still on bootstrap 3 that's why we went with react strap, it wont be nice to use both together especially one with different bootstrap impl, lets try to get it working with reactstrap, the features in react need not be pixel perfect in sync with one in angular, we can make changes where it makes sense